python/graphstorm/gconstruct/file_io.py

```diff
@@ -200,15 +200,15 @@ def read_data_parquet(data_file, data_fields=None):
     """
     table = pq.read_table(data_file)
     data = {}
-    df_table = table.to_pandas()
+    df_table: pd.DataFrame = table.to_pandas()
```

Comment:
Let keep it as it was. 
We do not apply strong typing in graphstorm code right now.